lesson1/homework#1/promise.js

```diff
@@ -0,0 +1,27 @@
+const XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest;
+
+const promiseArr = [];
+
+makeGetRequest = url => {
+  return new Promise(function(resolve, reject) {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url, true);
```

Comment:
```suggestion
    xhr.open('GET', url);
```
true выставляется по умолчанию:)